collect_app/src/main/java/org/odk/collect/android/application/initialization/ApplicationInitializer.kt

```diff
@@ -31,10 +28,7 @@ class ApplicationInitializer(
     private val userAgentProvider: UserAgentProvider,
     private val propertyManager: PropertyManager,
     private val analytics: Analytics,
-    private val upgradeInitializer: UpgradeInitializer,
-    private val analyticsInitializer: AnalyticsInitializer,
-    private val projectsRepository: ProjectsRepository,
-    private val settingsProvider: SettingsProvider
+    private val initializers: List<Initializer>
```

Comment:
We've talked before about not wanting to use the Jetpack App Initialization framework and just continue to develop our own custom approach. I wanted to move to using a list of initializers here rather than passing another explicit one in - this made it easier to add the new initializer and it'll make it easy to add others in the future.